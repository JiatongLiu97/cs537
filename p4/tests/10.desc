[8] Test the case that one on-queue page is deallocated with CLOCKSIZE = 8
