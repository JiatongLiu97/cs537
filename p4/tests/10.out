XV6_TEST_OUTPUT Index 0: pdx: 0x0, ptx: 0x8, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 1: pdx: 0x0, ptx: 0x7, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 2: pdx: 0x0, ptx: 0x6, writable bit: 1, encrypted: 0, ref: 0
